packages/chayns-toolkit/src/cli.ts

```diff
@@ -52,4 +53,14 @@ program
 		console.info("")
 	})
 
+program
+	.command("test")
+	.description("tests your code for possible errors using jest")
+    .option("-w, --watch", "watch for changes to rerun tests", false)
+    .option("--setupFile <path>", "file that should be executed before the tests are initialized", '')
```

Comment:
What use-case did you have in mind with this option? Just so we don't include an option that nobody uses